CMakeLists.txt

```diff
@@ -2,10 +2,11 @@ cmake_minimum_required(VERSION 3.5)
 project(obs-v4l2sink)
 
 
-include(external/FindLibObs.cmake)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/external)
```

Comment:
I've dropped that completely, external/FindLibObs.cmake isn't useful if the obs-studio-devel is installed as appropriate.